### ISSUE TITLE
LOR S5 Motion effects fail to import

### ISSUE THREAD

**Issue by gergmchairy** [smeighan/xLights#2206]
Hi Keith,

As per your comment on facebook, please find attached issue.
Trying to import motion effects from LOR S5, nothing imports on the selected channels.

eg Arches 1-4 in the attached sequence - -amongst other.
Xlights 2020.34

Cheers!

Greg
[https://www.dropbox.com/s/znklyl8a0omzqke/2020%20WRIGHT%20A%20Mad%20Russians%20Christmas%20MLC1.loredit?dl=0](url)

**Comment by keithsw1111** [comment]
I have downloaded and imported. There were a number of issues the biggest being that the data in the file is slightly different that other files we have seen but I have managed to adapt for the slight differences.

You are also using effects we have never seen before. If you want to help us improve the import some screen shots of the effect settings (including which prop and time would help us try to work out how to bring the effects in more completely. There are also effects like singleblock I have never seen before. I have assumed it maps to our single line but I dont know what the settings are so it just comes in as an effect ... you will need to do all the fine tuning.

Updates will be in .36
